UPGRADE.md

```diff
@@ -184,6 +184,12 @@ And you may remove your Mix configuration file:
 rm webpack.mix.js
 ```
 
+To wrap up, you may remove your Mix build manifest as well as the rule from the `.gitignore` config file:
```

Comment:
The Mix manifest isn't included in the default `.gitignore`. I know it's very common though, in addition to ignoring `public/css`, `public/js`, etc. so if we mention this maybe we should also mention removing the old build files and ignores?

Reply:
With Vite, the build assets and manifest are placed in `public/build` rather than `public/css`, `public/js`, etc.

I've added `public/build` to the `.gitignore` for `laravel/laravel` so compiled assets will now be ignored by default.

Reply:
Right! I ended up removing this. People will figure this out on their own 👍